src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -876,6 +886,8 @@ def __init__(self, config):
         self.roberta = RobertaModel(config, add_pooling_layer=False)
         self.lm_head = RobertaLMHead(config)
 
+        self.update_keys_to_ignore(config, ["lm_head.decoder.weight"])
```

Comment:
Could you add a comment above mentioning why we're doing this? I'm offering a suggestion.
```suggestion
        # The LM head weights require special treatment only when they are tied with the word embeddings
        self.update_keys_to_ignore(config, ["lm_head.decoder.weight"])
```